redbot/cogs/filter/filter.py

```diff
@@ -175,14 +198,35 @@ async def _filter_channel(self, ctx: commands.Context):
         """
         pass
 
+    @_filter_channel.command(name="clear")
+    async def _channel_clear(self, ctx: commands.Context):
+        """Clears this channels filter list."""
+        channel = ctx.channel
+        author = ctx.author
+        filter_list = await self.config.channel(channel).filter()
+        if not filter_list:
+            return await ctx.send(_("The filter list for this channel is empty."))
+        await ctx.send(_("Are you sure you want to clear this channels's filter list? (yes/no)"))
```

Comment:
"yes/no" is not conditional on the used language so should not be part of the translated text:
```suggestion
        await ctx.send(
            _("Are you sure you want to clear this channel's filter list?") + " (yes/no)"
        )
```